src/Components/Browser.JS/src/Boot.Server.ts

```diff
@@ -5,11 +5,19 @@ import { MessagePackHubProtocol } from '@aspnet/signalr-protocol-msgpack';
 import { OutOfProcessRenderBatch } from './Rendering/RenderBatch/OutOfProcessRenderBatch';
 import { internalFunctions as uriHelperFunctions } from './Services/UriHelper';
 import { renderBatch } from './Rendering/Renderer';
-import { fetchBootConfigAsync, loadEmbeddedResourcesAsync } from './BootCommon';
+import { fetchBootConfigAsync, loadEmbeddedResourcesAsync, autoStartIfApplicable } from './BootCommon';
 import { CircuitHandler } from './Platform/Circuits/CircuitHandler';
 import { AutoReconnectCircuitHandler } from './Platform/Circuits/AutoReconnectCircuitHandler';
 
-async function boot() {
+let started = false;
+
+async function boot(options?: any) {
+
+  if (started) {
+    throw new Error('Blazor has already started.');
+  }
+  started = true;
+
   const circuitHandlers: CircuitHandler[] = [ new AutoReconnectCircuitHandler() ];
```

Comment:
I took no steps to address the fact that we still have mutable global state for other things like reconnect. LMK if you think that fits in nicely with this PR.

Reply:
If you wanted to refactor those other pieces to match your new pattern then I'm sure that would work well, but if you prefer to consider that a separate future work item that would be fine too.

Reply:
Lets discuss at standup. I think it's somewhat worthwhile to try and change the pattern we're using once. If this PR is going to introduce the `start()` function that it might make sense to establish the pattern we want to use for everything in this preview.

Reply:
I'd be open to doing an all-over cleanup pass over the `.JS` project after preview 4. I'm not sure we'll have time to change all the public-facing JS APIs prior to preview 4 lockdown, but I guess it depends on exact cutoff dates.

Reply:
We're slowing down feature developement **so that** we can polish things that need polishing. Anything that's part of the making the experience consistent and well-thought-through is fair game. 

We could just as well decide to punt this particular item.